src/main.py

```diff
@@ -1,13 +1,18 @@
 from fastapi import FastAPI
 import logging.config
 from src.config import settings
+from src.database import get_session
 
 
 logging.config.fileConfig('logging.conf')
 
 
 app = FastAPI()
 
+session = get_session()
```

Comment:
why did you add these lines here?
read about dependency_infection in fastapi, you will use it later in yourendpoints as db_session